plugins/dap/dap-python.nix

```diff
@@ -0,0 +1,81 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.dap.extensions.dap-python;
+  helpers = import ../helpers.nix {inherit lib;};
+in {
+  options.plugins.dap.extensions.dap-python = {
+    enable = mkEnableOption "dap-python";
+
+    package = helpers.mkPackageOption "dap-python" pkgs.vimPlugins.nvim-dap-python;
+
+    adapterPythonPath = mkOption {
+      default = "python3";
```

Comment:
maybe set the default to `${pkgs.python3)/bin/python` so that it is available by default ?